src/server.c

```diff
@@ -5959,7 +5959,7 @@ void dismissClientMemory(client *c) {
     /* Release the reply array only if the average buffer size is bigger
      * than a page. */
     if (listLength(c->reply) &&
-        c->reply_bytes / listLength(c->reply) >= server.page_size)
+        c->reply_bytes/listLength(c->reply) >= server.page_size)
```

Comment:
I think the common style in redis is with spaces.
not very consistent though..

Reply:
actually, i am a bit tangled, in this case, `&&` is the top operator, i prefer to not use spaces in sub judgement